server/testdriver/core/pom.xml

```diff
@@ -68,5 +68,33 @@
          <artifactId>spymemcached</artifactId>
       </dependency>
    </dependencies>
+   <build>
+      <plugins>
+         <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-resources-plugin</artifactId>
+            <executions>
+               <execution>
+                  <id>copy-server-config</id>
+                  <phase>validate</phase>
+                  <goals>
+                     <goal>copy-resources</goal>
+                  </goals>
+                  <configuration>
+                     <outputDirectory>${project.basedir}/src/main/resources</outputDirectory>
```

Comment:
Please copy it to `target/generated-sources/`, so you don't have to add it to `.gitignore`

@tristantarrant isn't there a way to use the default server configuration from `server/runtime` without copying it?

Reply:
We don't want to carry around any version of the server configuration in the testdriver-core module. CONTAINER mode would use the default configs present in the image and EMBEDDED mode should only be used by our own integration tests

Reply:
ok